drivers/gpio/Kconfig

```diff
@@ -85,8 +85,6 @@ source "drivers/gpio/Kconfig.emul"
 
 source "drivers/gpio/Kconfig.psoc6"
 
-source "drivers/gpio/Kconfig.pcal6408a"
```

Comment:
same question here: what happened to these lines? are the branches not branched from the same point/commit?